src/ui/QuerySuggestPreview/QuerySuggestPreview.ts

```diff
@@ -22,6 +22,20 @@ export interface IQuerySuggestPreview {
   executeQueryDelay?: number;
 }
 
+/**
+ * This component adds previews of the top query results matching the currently focused query suggestion in the search box.
+ *
+ * As such, this component only works when an [`Omnibox`]{@link Omnibox} is present in the search interface.
+ *
+ * Moreover, this component can use a [result template](https://docs.coveo.com/en/413/) in its markup configuration to render previews with.
+ *
+ * **Example**
+ * ```
+ *   <div class="CoveoQuerySuggestPreview"></div>
```

Comment:
Does the component need to be nested inside the Omnibox? If so, I think it would be good to show in the example. If not, then how would the component know where to attach if there are multiple Omniboxes in the page?

Reply:
It doesn't need to be inside an `Omnibox`, and it would append its results to any Omnibox that hovered a suggestion. Internally, it feeds the `Omnibox` by handling an event thrown by `Omnibox`, meaning it would even be possible (although currently useless since there's no filters of any sort) to have multiple `QuerySuggestPreview`.

I'll update the documentation to add more details on how to use `QuerySuggestPreview`.

Reply:
Sometimes clients have multiple Omnibox components in the page. I wonder if they will want to be able to fine-tune which ones the `CoveoQuerySuggestPreview` affects? @olamothe, any thoughts?

Reply:
This could pretty easily be done.